### PR TITLE
fix: #tno-1590 - made duplicate detection more robust

### PR DESCRIPTION
**Pull request by kylgarmor**
- since we cant delete in non-PROD, we have to do it the long way

### CODE REVIEW

File: services/net/image/ImageAction.cs [hunk 1]
```diff
@@ -154,18 +156,35 @@ public ImageAction(IApiService api, IOptions<ImageOptions> options, ILogger<Imag
...
+                        Logger.LogDebug($"Image Ingest : ADD NEW : [{manager.Ingest.IngestType?.Name ?? "Image"} - {manager.Ingest.Name}] : {path}");
+                    } else {
+                        // if we already have a contentReference, check if this is actually an update to the image
+                        if (newReference.PublishedOn > reference.PublishedOn)
+                        {
+                            isUpdatedSourceContent = true;
+                            // update the existing refernce published on date with the new date
+                            reference.PublishedOn = newReference.PublishedOn;
+
+                            // What about the worst case scenario: one Editor changes it in MMI and another Editor changes it in TNO?
```
Comment (Fosol) [inline services/net/image/ImageAction.cs:170]: I don't think we care about this scenario.  In TNO they don't even look at front pages in the editor app.

File: services/net/image/ImageAction.cs
Comment (Fosol) [inline services/net/image/ImageAction.cs:1]: Have you run your linter for nullability?

File: services/net/image/ImageAction.cs (hunk 1 above)
Comment (kylgarmor) [reply services/net/image/ImageAction.cs:170]: that comment was acopy/paste from lesewhere.  not relevant

**Review by Fosol — APPROVED** [review]

File: services/net/image/ImageAction.cs
Comment (kylgarmor) [reply services/net/image/ImageAction.cs:1]: Ive fixed the two possible null issues in `ImageAction.cs`.  I installed SonarLint and it's mad about all sorts of stuff...

**Pull request merged**